src/main/java/com/mindera/school/spaceshiprent/controller/SpaceshipController.java

```diff
@@ -12,31 +16,39 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.validation.Valid;
+import java.time.LocalDate;
 import java.util.List;
 
+@Slf4j
 @RestController
 @RequiredArgsConstructor
 public class SpaceshipController {
 
     private final SpaceShipServiceImpl spaceShipService;
 
     @PostMapping("/spaceships")
-    public ResponseEntity<SpaceShipDetailsDto> createSpaceship(@RequestBody CreateOrUpdateSpaceshipDto dto){
+    public ResponseEntity<SpaceShipDetailsDto> createSpaceship(@RequestBody @Valid CreateOrUpdateSpaceshipDto dto){
+        log.info(LoggerMessages.POST_REQUEST, LoggerMessages.SPACESHIP, LocalDate.now());
```

Comment:
the logger already log the timestamp of the log, no need to wtite it in the message